test/unit/operations/subscribe-to-updates.test.ts

```
@@ -80,9 +351,72 @@ describe('SubscribeToUpdates', (): void => {
+            traceContext?: unknown;
+            externalTransactionHash?: unknown;
+            events: Array<{
+              CreatedEvent?: { representativePackageId?: string; acsDelta?: boolean; contractKey?: unknown };
+            }>;
+          };
+        };
+      };
+    };
+    // Normalized Zod output (not the raw wire object): null optionals → undefined.
+    expect(delivered).not.toBe(wireMessage);
+    const value = delivered.update.Transaction.value;
```

ESLint

[error] Use object destructuring.

(prefer-destructuring)